django_project/feti/api_views/occupation.py

```diff
@@ -0,0 +1,21 @@
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from feti.models.occupation import Occupation
+from feti.serializers.occupation_serializer import OccupationListSerializer
+
+__author__ = 'Alison Mukoma <alison@kartoza.com>'
+__date__ = '24/10/17'
+__license__ = "GPL"
+__copyright__ = 'kartoza.com'
+
+
+class OccupationListApiView(APIView):
+    """
+    Api to list all available occupations.
+    """
+    def get(self, request):
+        occupations = Occupation.objects.all()
```

Comment:
hi @sonlinux 
could you use elastic search instead like other API?

Reply:
hi @meomancer the API should return all available occupations, so I think that is why @sonlinux did not use elastic search

Reply:
Hi @meomancer, the idea was to have all theoccupations show without using elasticsearch.

Reply:
Hi @ann26 yes it would be okay if the other attributes won't be that much useful. Lets first hear what @meomancer is thinking about this.